<launch>

    <env name="ROSCONSOLE_FORMAT" value="${function}: ${message}"/>
    
    <!-- 指定可能なボーレート
        9600    
        57600   
        115200  
        1000000 
        2000000 
        3000000 
        4000000 
        4500000 
        10500000
     -->
    <arg name="DEVICE_NAME" default="/dev/ttyUSB0"/>
    <arg name="PREVIOUS" default="1000000"/>
    <arg name="AFTER"    default="3000000"/>

    <node pkg="dynamixel_handler" name="dxl_change_baudrate" type="dynamixel_change_baudrate_node" output="screen">
        <param name="device_name"    value="$(arg DEVICE_NAME)"/>
        <param name="prev_baudrate"  value="$(arg PREVIOUS)"/>
        <param name="after_baudrate" value="$(arg AFTER)"/>
        <param name="max_id"         value="45"/>
        <param name="min_id"         value="0"/>
    </node>

</launch>